front-end/src/app/reports/transactions/transaction-input/transaction-input.component.html

```diff
@@ -188,10 +188,12 @@ <h3>{{ transactionType.committeeCandidateHeader }}</h3>
     <ng-container *ngIf="transaction?.transactionType?.hasSupportOpposeCode()">
       <app-support-oppose-input [form]="form" [formSubmitted]="formSubmitted"></app-support-oppose-input>
     </ng-container>
-    <ng-container *ngIf="transaction?.transactionType?.hasCandidateInformation(form)">
+    <ng-container
+      *ngIf="transaction?.transactionType?.hasCandidateInformation(form) && !transactionType?.hidePrimaryContactLookup ">
```

Comment:
Good point. Renamed.

Reply:
Ah the reason that data wasn't showing was because I had the hidePrimaryContactLookup call in the wrong place. I wanted to only hide the Lookup part of the candidate information but accidentally hid the whole thing. I moved my check to the right location and it's showing up now. I also noticed it wasn't making those fields read only so I had to make some fixes for that.